### PR TITLE
Adds RITEway Unit Tests

### PR DESCRIPTION
**Pull request by jwicks31**
### Description of Changes

Adds Unit Test with RITEway for comparison with Alsatian and Tape.

### PR Review Checklist

* [ ] **Have you tested the changes?**
* [ ] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [ ] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [ ] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [ ] **Are the added/edited files named appropriately and located correctly in the project directory structure?**

### CODE REVIEW

File: src/components/atoms/Button/Button.test.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      given: 'text and true loading props',
+      actual: $('.Button').text(),
+      expected: 'loading'
+    });
+  }
+  {
+    const props = { text: 'test', loading: false }
+    const $ = dom.load(render(<Button {...props}/>))
+    assert({
+    given: 'text and false loading props',
```
Comment (warrenv) [inline src/components/atoms/Button/Button.test.tsx]: Can we add a 'should' line here?

File: src/components/atoms/Button/Button.test.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      actual: $('.Button').text(),
+      expected: 'test'
+    });
+  }
+  {
+    const props = { text: 'test', loading: true }
+    const $ = dom.load(render(<Button {...props}/>))
+    assert({
+      given: 'text and true loading props',
+      actual: $('.Button').text(),
```
Comment (warrenv) [inline src/components/atoms/Button/Button.test.tsx]: Can we add a 'should' line here?

File: src/components/atoms/Button/Button.test.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    should: 'render a button',
+    actual: $('.Button').length,
+    expected: 1
+    });
+  }
+  {
+    const props = { text: 'test'}
+    const $ = dom.load(render(<Button {...props}/>))
+    assert({
+      given: 'text prop',
```
Comment (warrenv) [inline src/components/atoms/Button/Button.test.tsx:25]: Can we add a 'should' line here?

**Comment by jwicks31** [comment]
Closing to add tests as single PR.

File: package.json [hunk 4]
```diff
@@ -10,6 +10,7 @@
     "serve": "ws --spa index.html -d ./dist",
     "stylelint": "stylelint './src/**/*.scss' ",
     "stylelint:fix": "stylelint './src/**/*.scss' --fix",
+    "test": "ts-node -r ignore-styles tests/**/*.ts",
```
Comment (jwicks31) [inline package.json]: One more thing that might help is installing `tsconfig-paths` for absolute imports in our components. Then add `-r tsconfig-paths/register`

**Comment by warrenv** [comment]
frost-api is the only other app using that package.
How about we do that in a separate PR?

**Pull request closed without merging**